classes/Upgrader.php

```diff
@@ -176,7 +176,7 @@ public function checkPSVersion($force = false)
     /**
      * load the last version informations stocked in base.
      *
-     * @return Upgrader
+     * @return $this
```

Comment:
```suggestion
     * @return static
```
No? I thought we used this rather than `$this` or `self`

Reply:
static is supported since PHP 8.0 (discussed with @PierreRambaud